src/encoded/tests/test_types_variant.py

```diff
@@ -228,3 +228,39 @@ def test_most_severe_location(
     ).json["@graph"][0]
     most_severe_location = patch_response.get("most_severe_location")
     assert most_severe_location == expected
+
+
+@pytest.mark.parametrize(
+    "hgvsc,hgvsp,result", [
+        (
+            'ENST00000643905.1:c.500_501del',
+            'ENSP00000495949.1:p.Val167GlyfsTer6',
+            ['c.500_501del', 'p.Val167GlyfsTer6', 'p.V167GfsTer6']
+        ),
+        (
+            'ENST00000643905.1:c.*1054C>T',
+            None,
+            ['c.*1054C>T']
+        ),
+        (
+            'ENST00000430580.6:c.2290C>A',
+            'ENSP00000474456.1:p.Gln764Lys',
+            ['c.2290C>A', 'p.Gln764Lys', 'p.Q764K']
+        ),
+        (
+            'ENST00000485362.1:n.41-3C>T',
+            None,
+            ['n.41-3C>T']
+        )
+    ]
+)
+def test_additional_variant_names(testapp, gene, variant, hgvsc, hgvsp, result):
+    variant_atid = variant.get("@id")
+    patch_body = {"genes": [{
+        "genes_most_sever_gene": gene.get("@id"),
```

Comment:
Typo here, but the field also isn't required for the calcprop, so consider deleting as well.
```suggestion
        "genes_most_severe_gene": gene.get("@id"),
```

Reply:
This isn't required, but it is part of the original variant fixture and would get removed without the line. It doesn't really matter either way, though.